src/utils/githubHelper.js

```diff
@@ -1,21 +1,28 @@
 import axios from 'axios';
 
-const githubAxios = axios.create({
-  baseURL: 'https://api.github.com',
-  headers: {
-    Accept: 'application/vnd.github.v3+json',
-    Authorization: `token ${process.env.REACT_APP_GITHUB_TOKEN}`
+const apiUrl = () => {
+  if(process.env.REACT_APP_BUNDLED_DEPLOY === 'true') {
+    return ""; // Uses relative path since client is being served from express server
+  } else if(process.env.REACT_APP_EXPRESS_URL) {
+    return process.env.REACT_APP_EXPRESS_URL; // Manually overridden URL
+  } else {
+    return "http://localhost:5000"; // Default URL for dev
   }
+}
+
+const githubAxios = axios.create({
+  baseURL: apiUrl()
 });
 
 export function searchGithub(searchTerm) {
-  return githubAxios.get(`/search/repositories?q=${searchTerm}`)
+  // These urls need encoded in the event they include special characters
+  return githubAxios.get(`/search?q=${encodeURIComponent(searchTerm)}`);
 }
 
 export function getContributing(repoName) {
-  return githubAxios.get(`/repos/${repoName}/contents/CONTRIBUTING.md`);
+  return githubAxios.get(`/contributing/${encodeURIComponent(repoName)}`);
 }
 
 export function getOpenIssues(repoName) {
-  return githubAxios.get(`/search/issues?q=repo:${repoName}+state:open+is:issue&per_page=15`);
+  return githubAxios.get(`/open_issues/${encodeURIComponent(repoName)}`);
```

Comment:
Just curious if you figured out something new and tasty for the issues query? That is all :)